.gitignore

```diff
@@ -0,0 +1 @@
+wss-wu-dryville.jpg
```

Comment:
Not required, but you will see that red "no entry" symbol if you don't have a new line at the end of the file. You can fix it by adding a new empty line (just hit enter), saving, committing, and pushing.